design-system/react/src/components/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from "../Box";
+import { Icon } from "../Icon";
+
+import type { MenuProps } from "./Menu";
+import { Menu } from "./Menu";
+
+export default {
+  component: Menu,
+  title: "UI/Menu",
+  argTypes: {},
+};
+
+export const Usage = (args: MenuProps) => (
+  <Box css={{ width: "200px" }}>
+    <Menu {...args} autoFocus disabledKeys={["edit"]} aria-label="Actions">
```

Comment:
any specific reason for having a `disabledKeys` prop in the Menu wrapper element?

My first impression is that would be better to inform disabled in the actual `Menu.Item`
for example
```
<Menu.Item key="settings" isDisabled>
  Settings
</Menu.Item>
```

Reply:
This is API already implemented in [React Aria](https://react-spectrum.adobe.com/react-aria/useMenu.html), I just followed the pattern to avoid reinventing/making complex things.

Reply:
It's not a good pattern but it's the way the lib uses it to manage all the interactions (arrow navigation, disable clicks etc), so nothing to do now I guess